.github/workflows/formatting.yml

```diff
@@ -0,0 +1,19 @@
+name: 'File Formatting with Prettier'
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '00 8 * * *'
```

Comment:
Please add a comment for how long this is

Reply:
It's 08:00am every morning, indefinitely.
Did you want it in the file?

Reply:
```suggestion
    - cron: '0 0 * * *'
```